app/components/log_search_component.rb

```diff
@@ -0,0 +1,9 @@
+class LogSearchComponent < ViewComponent::Base
```

Comment:
Same here

Reply:
WIP still will be refactored to use generic one.